rpc/test/client_test.go

```diff
@@ -85,16 +85,17 @@ func testBroadcastTxSync(t *testing.T, client rpc.HTTPClient) {
 //--------------------------------------------------------------------------------
 // query
 
-func testTxKV(t *testing.T) ([]byte, []byte, []byte) {
+func testTxKV(t *testing.T) ([]byte, []byte, types.Tx) {
 	k := randBytes(t)
 	v := randBytes(t)
-	return k, v, []byte(Fmt("%s=%s", k, v))
+	return k, v, types.Tx(Fmt("%s=%s", k, v))
 }
 
 func sendTx(t *testing.T, client rpc.HTTPClient) ([]byte, []byte) {
 	tmResult := new(ctypes.TMResult)
 	k, v, tx := testTxKV(t)
-	_, err := client.Call("broadcast_tx_commit", map[string]interface{}{"tx": tx}, tmResult)
+	txBytes := []byte(tx) // XXX
```

Comment:
Can we trace why this is the case?  It seem quite odd and maybe a bug in another level we should fix rather than work around.